debianpkg/rules

```diff
@@ -136,18 +136,12 @@ override_dh_auto_configure:
 	fi
 
 override_dh_auto_build:
-	#dh_auto_build
-	$(MAKE)
-	dh_auto_build -- -C doc draft-zebra-00.txt
-
-
 	# doc/ is a bit crazy
 ifeq ($(GENERATE_PDF), 1)
-	dh_auto_build -- -C doc frr.pdf || true # pdfetex fails with exit code 1 but still produces a good looking .pdf
+	dh_auto_build -- -C doc user.pdf
```

Comment:
misspelling? Should be `FRR.pdf` - not `user.pdf`
`dh_auto_build -- -C doc FRR.pdf`